### DISCUSSION TITLE
Notification sounds during osu!lazer multiplayer gameplay

### DISCUSSION THREAD

**Discussion by cj4m** [ppy/osu#15090] [Q&A] [answered]
I've been playing a lot of multiplayer lately, and had noticed a few odd sounds playing at random which made me realize it wasn't a hitsound my skin was missing. At first I thought it was chat messages but didn't see anything during the break period, after [mentioning it to peppy on twitter](https://twitter.com/ppy/status/1448457443319508994) and he had guessed it was players joining/leaving the multiplayer lobby. I'm also not sure if there's sounds during leaderboard position changes or if I just heard notification sounds playing as I threw a couple errors when importing maps from stable. 
I will try to get a video of this tomorrow when multiplayer is a bit busier :) 

On the note of importing beatmaps (and I can move to different discussion if needed), is there any way to pause the import process during gameplay? or is cancelling it the only option available? i'm still surprised how great gameplay is during the frame drops caused by importing

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15092
